test_schemas/en/test_checkbox.json

```diff
@@ -32,6 +32,67 @@
             "groups": [
                 {
                     "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "null-label-checkbox",
```

Comment:
`null` is how we've implemented the schema for this, but as a feature I think `no label` is clearer and more appropriate.